x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -303,6 +303,23 @@
                         }
                     }
                 },
+                "reporting": {
+                    "properties": {
+                        "jobtype": {
+                            "type": "keyword"
+                        },
+                        "status": {
+                            "type": "keyword"
+                        },
+                        "csv": {
+                            "properties": {
+                                "byteLength": {
+                                    "type": "long"
+                                }
+                            }
+                        }
+                    }
+                },
```

Comment:
From the code, I'm finding it a bit tough to reason about the kinds of questions this data will help us answer. My understanding is:

We will have a way to create a sequence of linked events for when something (like CSV download) starts, how long it takes to complete, when it errors.

How does the CSV byte length factor in?

Reply:
Hi JL! Collecting this type of metric information is desired for [Monitoring of Reporting](https://github.com/elastic/kibana/issues/29281). Basically, it will allow us answer questions like: the average time needed to create a CSV report that is X bytes. When we see that info side by side with other charts like the RAM consumed over time, it will let us see how different factors play together.

There are more metrics that can be added, but perhaps don't need to be in this PR:
 - the `http.max_content_length` cluster setting in ES
 - how many scroll operations were done to generate the CSV
 - what was the average byte size of each scroll response

This was unclear to you because there is not a good enough description of the work done for this PR. The task description is also lacking. I will update a few things on the project board to fix this.

Reply:
@jloleysens since writing the last comment, I've added more to the description of this PR. All of the additions were copied from the project spec document (internal link).

Reply:
@pmuellr based on your advice, it's probably best to add a small number of fields in this initial PR.